save-frontend/src/main/kotlin/com/saveourtool/save/frontend/App.kt

```diff
@@ -210,6 +211,11 @@ class App : ComponentWithScope<PropsWithChildren, AppState>() {
                                     }
                                 }
 
+                                Route {
+                                    path = "/${FrontendRoutes.CONTESTS_GLOBAL_RATING.path}"
```

Comment:
@Arrgentum will need to support this route in #1088 if we merge this one earlier

Reply:
OK